src/limbo/bayes_opt/boptimizer.hpp

```diff
@@ -35,18 +35,18 @@ namespace limbo {
         /**
         The classic Bayesian optimization algorithm.
 
-        \rst
+        \\rst
```

Comment:
Why this? It seems the documentation generation was working on this file! see: http://www.resibots.eu/limbo/api.html#_CPPv2N5limbo9bayes_opt10BOptimizerE